user_manual/files/webgui/custom_groups.rst

```diff
@@ -9,6 +9,11 @@ This wasn't the most efficient way to work.
 To address that, as of ownCloud 10.0, you can now create your own groups on-the-fly, through a feature called "Custom Groups". 
 Here's how to use it. 
 
+.. note::
+   	- creating a customgroup with an existing display name displays an error
+	- renaming a customgroup to an existing display name displays an error
```

Comment:
Proposal ``an existing display name displays an error`` --> ``an existing display name shows an error``
to avoid the double use of "display"